core/tests/sync_service_unsynced_client_tests.rs

```diff
@@ -78,7 +78,7 @@ fn delete() {
     core.delete_file(doc.id).unwrap();
     assert::all_paths(&core, &["/"]);
     assert::all_document_contents(&core, &[]);
-    assert::local_work_paths(&core, &["/document"]);
+    assert::local_work_paths(&core, &[]);
```

Comment:
new deleted documents, which already weren't synced, are now not considered work